Makefile

```diff
@@ -15,6 +14,8 @@ dev:
 	$(eval RESOURCE_NAME_PREFIX=s165d01)
 	$(eval ENV_SHORT=dv)
 	$(eval ENV_TAG=dev)
+	$(eval AZURE_BACKUP_STORAGE_ACCOUNT_NAME=s165d01getaniddbbackupdv)
```

Comment:
do we need to back up dev database?

Reply:
Kept it just for local testing in `dev` when needed. But we can add it later when needed. so removed it.